packages/compass-indexes/src/components/search-index-template-dropdown/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { ATLAS_SEARCH_TEMPLATES } from '@mongodb-js/mongodb-constants';
+import type { SearchTemplate } from '@mongodb-js/mongodb-constants';
+import React, { useState, useCallback } from 'react';
+import {
+  Select,
+  Option,
+  Icon,
+  css,
+  spacing,
+  Tooltip,
+} from '@mongodb-js/compass-components';
+
+const dropdownStyles = css({
+  display: 'flex',
+  overflow: 'hidden',
+  gap: 0,
+});
+
+const overlapDropdownLabelStyles = css({
+  marginLeft: -spacing[3],
+});
+
+type SearchIndexTemplateDropdownProps = {
+  tooltip: string;
+  onTemplate: (template: SearchTemplate) => void;
+};
+
+export const SearchIndexTemplateDropdown: React.FunctionComponent<
+  SearchIndexTemplateDropdownProps
+> = ({ tooltip, onTemplate }) => {
+  const [templateValue, setTemplateValue] = useState('0');
+
+  const onChooseTemplate = useCallback(
+    (value: string) => {
+      setTemplateValue(value);
+      onTemplate(ATLAS_SEARCH_TEMPLATES[+value]);
+    },
+    [onTemplate]
+  );
+
+  return (
+    <section className={dropdownStyles}>
+      <Select
+        value={templateValue}
+        allowDeselect={false}
+        onChange={onChooseTemplate}
+        label={'Template'}
+      >
+        {ATLAS_SEARCH_TEMPLATES.map((template, idx) => (
+          <Option key={idx} value={`${idx}`}>
+            {template.name}
+          </Option>
+        ))}
+      </Select>
+      <Tooltip
+        align="right"
+        triggerEvent="hover"
+        trigger={({ children, ...props }) => (
+          <>
+            {children}
```

Comment:
The tooltip will be aligned with the parent whatever place the children are rendered, because you are rendering this in the fragment, it just randomly aligns itself with the section here, is this expected?

![image](https://github.com/mongodb-js/compass/assets/5036933/ed6b0dad-ea79-4350-acc2-36766398e721)

Reply:
Did you change anything in the code?  The alignment should be on the right side.

Taking a look at the alignment though.

Reply:
Oh, understood. With the changes I've done, as you suggested, now it aligns correctly.

<img width="636" alt="Screenshot 2023-09-22 at 12 11 50" src="https://github.com/mongodb-js/compass/assets/1822138/359bfff9-09f0-4bda-876c-2932c10bac59">